tests/test_lora_inference.py

```diff
@@ -0,0 +1,42 @@
+"""
+Run this test in Lora adpater checking:
+
+```shell
+python3 test_lora_inference.py --prompt "Your prompt" --lora_path "path/to/lora" --lora_name "lora_adapter" --output_file "output.mp4" --fps 8
```

Comment:
Maybe we use a working command as an example here? 

Path to LoRA can either be local or a Hub id.